expression/builtin_time_vec_test.go

```diff
@@ -142,6 +142,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 			},
 		},
 	},
+	ast.Week: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime, types.ETInt}},
```

Comment:
The argument arg1 (field mode) is valid when it is 0-7 (more detail can see weekMode function) . I think we should add some test cases for it.